glossary/Yarn.md

```diff
@@ -0,0 +1,4 @@
+### Yarn
+
+Yarn is a package manager for the JavaScript programming language.
```

Comment:
maybe add "by facebook" here?

Reply:
Honestly, I prefer not to say who is behind what as packages have a slighttendency to change ownership over time and it might get outdated or sound like unnecessary promo. I'm thinking we can link to some external content from the glossary like said packages etc. so it will be obvious to people what is what.

Reply:
Currently it only works with JS. Plus, this is a JS-specific repo, so we can quietly omit that information at least until this is a stable feature.

Reply:
In that case it might be better as

> Yarn is a package manager made by Facebook. It can be used as an alternative to the npm package manager and is compatible with the public NPM registry.

reason the second NPM is capitalized: It's actually the company name, no the tool